src/styles/global.css

```
@@ -1777,6 +1787,70 @@ input:focus-visible + .custom-check {
+.settlement-feedback-error {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 12px;
+}
+
+.settlement-feedback-error button,
+.settlement-content-error button {
+  flex: 0 0 auto;
+  padding: 6px 8px;
+  border: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)